actioncable/lib/action_cable/engine.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails/engine'
+require 'active_support/ordered_options'
+require 'action_cable/helpers/action_cable_helper'
+
+module ActionCable
+  class Engine < ::Rails::Engine
+    config.action_cable = ActiveSupport::OrderedOptions.new
+
+    config.to_prepare do
+      ApplicationController.helper ActionCable::Helpers::ActionCableHelper
```

Comment:
Yes, that makes more sense. Should be available everywhere.

On Mon, Dec 14, 2015 at 5:24 PM, Rafael França <notifications@github.com>
wrote:

> In actioncable/lib/action_cable/engine.rb
> <https://github.com/rails/rails/pull/22585#discussion_r47518344>:
>
> > @@ -0,0 +1,27 @@
> > +require 'rails/engine'
> > +require 'active_support/ordered_options'
> > +require 'action_cable/helpers/action_cable_helper'
> > +
> > +module ActionCable
> > +  class Engine < ::Rails::Engine
> > +    config.action_cable = ActiveSupport::OrderedOptions.new
> > +
> > +    config.to_prepare do
> > +      ApplicationController.helper ActionCable::Helpers::ActionCableHelper
>
> Should we include in ActionView::Base?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/22585/files#r47518344>.
>